src/leap/keymanager/__init__.py

```diff
@@ -185,34 +190,46 @@ def _key_class_from_type(self, ktype):
             lambda klass: klass.__name__ == ktype,
             self._wrapper_map).pop()
 
-    def _get(self, uri, data=None):
+    @defer.inlineCallbacks
+    def _get_json(self, address):
```

Comment:
I have the feeling that we can come up with a better method name. `_get_key_from_nicknym`?